src/registration/CommunicationPreferencesCheckbox.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { LabeledCheckbox } from 'src/components/common';
+
+interface CommunicationPreferencesCheckboxProps {
+  title: string;
+  value: boolean;
+  setFunc: React.Dispatch<React.SetStateAction<boolean>> | undefined;
+}
+
+export const CommunicationPreferencesCheckbox = (props: CommunicationPreferencesCheckboxProps) => (
```

Comment:
Our convention has been to add a return type to component functions.

```suggestion
export const CommunicationPreferencesCheckbox = (props: CommunicationPreferencesCheckboxProps): ReactNode => (
```

Reply:
Also, style nit: generally, I'd suggest always using block bodies for functions.

```ts
const Component = () => {
  return (
    ...
  )
}
``` 

If you ever need to add logic before the return in the future, it's easier with this style.

Otherwise, changing from the expression body to the block body creates a large formatting diff.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#function_body